File: src/main/java/src/SistemaGestionBiblioteca/Libro.java
package SistemaGestionBiblioteca;

public class Libro {
    private int id;
    private String titulo;
    private String autor;
    private boolean prestado;

    public Libro(int id, String titulo, String autor) {
        this.id = id;
        this.titulo = titulo;
        this.autor = titulo;
        this.prestado = false;
    }

    public int getId() {
        return id;
    }

    @Override
    public String toString() {
        return "Libro{" +
                "id=" + id +
                ", titulo='" + titulo + '\'' +
                ", autor='" + autor + '\'' +
                ", prestado=" + prestado +
                '}';
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitulo() {
        return titulo;
    }

    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }

    public String getAutor() {
        return autor;
    }

    public void setAutor(String autor) {
        this.autor = autor;
    }

    public boolean isPrestado() {
        return prestado;
    }

    public void setPrestado(boolean prestado) {
        this.prestado = prestado;
    }
}

File: src/main/kotlin/SistemaGestionBiblioteca/Libro.kt
package org.example.SistemaGestionBiblioteca

class Libro(
    var id: Int,
    var titulo: String,
    var autor: String
) {
    var prestado: Boolean = false

    fun isPrestado(): Boolean {
        return prestado
    }

    fun switchPrestado(prestado: Boolean) {
        this.prestado = prestado
    }

    override fun toString(): String {
        return "Libro{" +
                "id=" + id +
                ", titulo='" + titulo + '\'' +
                ", autor='" + autor + '\'' +
                ", prestado=" + prestado +
                '}'
    }
}